docs/superpowers/plans/2026-08-18-target-aware-oidc-provisioning.md

```
@@ -0,0 +1,308 @@
+    no_origins_for_target "${#MARTYROLOGY_FRONTEND_URLS[@]}" "$TARGET" \
+        "Martyrology has no staging deployment yet. Use --target local" \
+        "(localhost, against a local Zitadel) or --target production." && return 0
+```
+
+Its `case "$TARGET"` block at `:799` already matches the required shape — leave it.
+
+- [ ] **Step 7: Document the two-sweep consequence in `usage()`**
+
+Add per-target lines for the three actions (precedent at `:99`), plus:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)